src/haliax/nn/linear.py

```diff
@@ -25,7 +24,13 @@ class Linear(eqx.Module):
 
     @staticmethod
     def init(
-        In: AxisSpec, Out: AxisSpec, *, key, use_bias=True, out_first: bool = False, dot_general=None
+        In: AxisSpec,
+        Out: AxisSpec,
+        *,
+        key,
+        use_bias=True,
+        out_first: bool = False,
```

Comment:
Nit: this seems like it only affects the internals... for the purpose of simplifying the torch integration, would it be harmful to make `out_first` default to True (or remove/ignore it entirely and always output in torch compatible mode?).

If it's a performance issue, maybe transform to the out_first=False mode during loading?

Reply:
It's more that GPT2's linear was out_first=False and I modeled Haliax's linear on that. I need to go through and make the default True...